src/chat/functions/sendFileMessage.ts

```diff
@@ -266,13 +268,20 @@ export async function sendFileMessage(
 
   if (options.type === 'audio') {
     rawMediaOptions.isPtt = options.isPtt;
+
+    // Definir isViewOnce apenas se isPtt for true
```

Comment:
Remove esse comentário por favor, o código é auto explicativo, nesse caso, quando for colocar novos comentários, coloca em ingles mesmo, mas esse pode remover.